dev-libs/date/date-3.0.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A date and time library based on the C++11/14/17 <chrono> header"
+HOMEPAGE="https://github.com/HowardHinnant/date"
+SRC_URI="https://github.com/HowardHinnant/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="only-c-locale test"
+RESTRICT="!test? ( test )"
+
+src_prepare() {
+	cmake_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_TZ_LIB=ON
+		-DUSE_SYSTEM_TZ_DB=ON
+		-DENABLE_DATE_TESTING=$(usex test)
+		-DCOMPILE_WITH_C_LOCALE=$(usex only-c-locale)
+	)
+	cmake_src_configure
+}
+
+src_test() {
+	cd "${BUILD_DIR}" || die
```

Comment:
Verbatim: "One can run tests by cd'ing into the test subdirectory and running testit. [...]"
	cd "${BUILD_DIR}" || die
	testit || die